docs/courses/classes.md

```diff
@@ -1042,7 +1043,7 @@ main()
 #!/bin/bash
 set -e
 POINTS=$(( ( RANDOM % 100 )  + 1 ))
-curl --retry 3 -s "$CODIO_AUTOGRADE_V2_URL" -d grade=$POINTS -d format=md -d feedback=test
+curl --retry 3 -s "$CODIO_AUTOGRADE_V2_URL" -d grade=$POINTS -d format=txt -d feedback='Simple text here'  -d extra_credit=$POINTS
```

Comment:
This is a terrible example -- it sets extra credit to the same value as the grade.

Please use a more realistic example